packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/Choices/Choices.spec.js

```diff
@@ -101,6 +101,73 @@ describe('Component > Choices', function () {
         }
       })
     })
+
+    it('should call handleDelete with choice ID on ChoiceButton backspace keyDown', function () {
+      const choiceId = 'FR'
+      const backspaceEventMock = { key: 'Backspace', preventDefault: sinon.spy(), stopPropagation: sinon.spy() }
+
+      const fireChoiceButton = wrapper.find(ChoiceButton).filterWhere(button => button.key() === choiceId)
+      fireChoiceButton.simulate('keydown', choiceId, backspaceEventMock)
+
+      expect(handleDeleteSpy).to.have.been.calledOnceWith(choiceId)
+    })
+  })
+
+  describe('with autoFocus', function () {
+    let wrapper
+
+    before(function () {
+      wrapper = mount(
+        <Choices
+          autoFocus
+          filteredChoiceIds={mockTask.choicesOrder}
+          task={task}
+        />, {
+          wrappingComponent: Grommet,
+          wrappingComponentProps: { theme: zooTheme }
+        }
+      )
+    })
+
+    it('should have first choice with hasFocus true and tabIndex 0', function () {
+      const firstChoiceButton = wrapper.find(ChoiceButton).first()
+      expect(firstChoiceButton.props().hasFocus).to.be.true()
+      expect(firstChoiceButton.props().tabIndex).to.equal(0)
+      expect(firstChoiceButton.props().choiceId).to.equal('RDVRK')
+    })
+
+    it('should have not first choices with hasFocus of false and tabIndex -1', function () {
```

Comment:
I having trouble understanding what this is saying here. Maybe "should not have focus on the other choices"?